cmd/flags.go

```diff
@@ -91,7 +91,7 @@ func addSigningAlgFlag(fs *pflag.FlagSet, alg *string, value string) {
 }
 
 func addClaimsFileFlag(fs *pflag.FlagSet, file *string) {
-	fs.StringVarP(file, "claims-file", "", "", "set path of JSON file containing optional claims (see: https://openpolicyagent.org/docs/latest/management/#bundle-signature-format)")
+	fs.StringVarP(file, "claims-file", "", "", "set path of JSON file containing optional claims (see:  https://openpolicyagent.org/docs/latest/management/#signature-format)")
```

Comment:
[nit] Can you drop the extra space, by any chance? (I don't suppose that it's a display issue with github only; if so, ignore me 😄)

Reply:
Ah, yep sorry about that, just pushed up the fix 😄